src/pysimfrac/src/general/simFrac.py

```diff
@@ -7,7 +7,7 @@
 """
 SimFrac object class. 
 """
-import os, sys
```

Comment:
Why was this line removed?

Reply:
Javier told me and I checked that it's not used in the script

Reply:
We use os and sys throughout the class for file manipulation and error checking. Unless we have run all the examples and confirmed those work without these, I vote to leave it in. Also, it's not an essential change from the new 2D source code. So, this would be a pull request with multiple facets, which is not best practices.